.github/CODEOWNERS

```diff
@@ -34,6 +34,10 @@
 /code/controllers/subsystem/dbcore.dm @Jordie0608
 /tools/SQLAlertEmail/ @Jordie0608
 
+# Kylerace
+/code/controllers/subsystem/spatial_gridmap.dm @Kylerace
+/code/__DEFINES/spatial_gridmap.dm @Kylerace
```

Comment:
I am assuming absolute pedantry 
```suggestion
# Kylerace
/code/__DEFINES/spatial_gridmap.dm @Kylerace
/code/controllers/subsystem/spatial_gridmap.dm @Kylerace
```
Truly I am a noble and useful maintainer, yes sir